dist/windows/installer.nsh

```diff
@@ -112,6 +112,13 @@ SectionEnd
 ;--------------------------------
 
 Function .onInit
+  ; create a mutex ensures only one installer is running
+  System::Call 'kernel32::CreateMutexA(i 0, i 0, t "qbt_installer") i .r1 ?e'
```

Comment:
I would suggest dropping `?e` and use `IntCmpU` to compare the returned value manually.
Just curious, is this code generated by AI?